apps/storybook/src/feedback/Toast.stories.tsx

```diff
@@ -1,118 +1,107 @@
+        "top",
+        "top-left",
+        "top-right",
+      ],
+    },
+  },
   args: {
-    children: "This is a default toast message",
-    position: "bottom-right",
-    type: "info",
+    children: <ToastTitle>This is an example toast message.</ToastTitle>,
   },
-  render: ToastTemplate,
-};
+  component: Toast,
+  render: function Render({ position, ...args }) {
+    const [open, setOpen] = useState(false);
+
+    const handleOpen = () => setOpen(true);
+    const handleClose = () => setOpen(false);
+    return (
+      <Flex flexDirection="column" gap="4">
+        <Button onClick={handleOpen}>Show Toast</Button>
 
-export const SuccessToast: Story = {
+        <Toaster position={position}>
+          {open && <Toast {...args} onOpenChange={handleClose} />}
+        </Toaster>
+      </Flex>
+    );
+  },
+} as Meta<StoryProps>;
+
+type Story = StoryObj<StoryProps>;
+
+export const Basic: Story = {};
+
+export const Success: Story = {
   args: {
-    children: "Operation completed successfully!",
-    position: "top-left",
-    type: "success",
+    colorScheme: "success",
   },
-  render: ToastTemplate,
 };
 
-export const WarningToast: Story = {
+export const Warning: Story = {
   args: {
-    children: "Please be cautious of this action.",
-    position: "bottom-left",
-    type: "warning",
+    colorScheme: "warning",
   },
-  render: ToastTemplate,
 };
 
-export const DangerToast: Story = {
+export const Danger: Story = {
   args: {
-    children: "An error occurred. Please try again.",
-    position: "top",
-    type: "danger",
+    colorScheme: "danger",
   },
-  render: ToastTemplate,
 };
 
-export const CustomPositionedToast: Story = {
+export const Position: Story = {
   args: {
-    children: "This toast appears in the top-left corner.",
     position: "top-left",
-    type: "info",
   },
-  render: ToastTemplate,
 };
 
-export const WithAction: Story = {
+export const Action: Story = {
   args: {
-    action: "Undo",
-    children: "This toast is with an action.",
-    type: "info",
+    children: (
+      <>
+        <ToastTitle>This is an example toast message.</ToastTitle>
+        <ToastAction altText="Undo" onClick={action("undo")}>
+          Undo
+        </ToastAction>
+      </>
+    ),
   },
-  render: ToastTemplate,
 };
 
-export const LongContentToast: Story = {
+export const LongContent: Story = {
   args: {
-    children:
-      "This is a toast with a longer message. It demonstrates how the toast component handles more content and potentially wraps text to multiple lines.",
-    position: "bottom",
-    type: "info",
+    children: (
+      <ToastTitle>
+        This is an example toast message that should span two lines.
+      </ToastTitle>
+    ),
   },
-  render: ToastTemplate,
 };
 
-export const InteractiveTest: Story = {
-  args: {
-    children: "This is an interactive test toast message",
-    position: "bottom-right",
-    type: "info",
-  },
+export const Interactive: Story = {
```

Comment:
Need to change the `text content` as its different from the default text.

Reply:
addressed